app/lib/actions/remote_execution/run_hosts_job.rb

```diff
@@ -104,6 +120,10 @@ def humanized_input
       def humanized_name
         '%s:' % _(super)
       end
+
+      def proxy_batch_size
+        Setting['foreman_tasks_proxy_batch_size']
```

Comment:
In theory this could lead to odd behaviours if the setting changed while the action is running. Would you be open to storing the batch size that is set in settings at the start of the action and then using that value till the end?

Reply:
And right away realized it's useless, so I've put it into `input` :)